types/node/events.d.ts

```diff
@@ -294,6 +294,35 @@ declare module 'events' {
          * @since v15.2.0, v14.17.0
          */
         static getEventListeners(emitter: _DOMEventTarget | NodeJS.EventEmitter, name: string | symbol): Function[];
+        /**
+         * Returns the currently set max amount of listeners.
+         *
+         * For `EventEmitter`s this behaves exactly the same as calling `.getMaxListeners` on
+         * the emitter.
+         *
+         * For `EventTarget`s this is the only way to get the max event listeners for the
+         * event target. If the number of event handlers on a single EventTarget exceeds
+         * the max set, the EventTarget will print a warning.
+         *
+         * ```js
+         * import { getMaxListeners, setMaxListeners, EventEmitter } from 'node:events';
+         *
+         * {
+         *   const ee = new EventEmitter();
+         *   console.log(getMaxListeners(ee)); // 10
+         *   setMaxListeners(11, ee);
+         *   console.log(getMaxListeners(ee)); // 11
+         * }
+         * {
+         *   const et = new EventTarget();
+         *   console.log(getMaxListeners(et)); // 10
+         *   setMaxListeners(11, et);
+         *   console.log(getMaxListeners(et)); // 11
+         * }
+         * ```
+         * @since v19.9.0
```

Comment:
this one (and another one) are newer, than version in PR (and header). Does it matter?

Reply:
This should have been added in v20.0.0 typings but was missed.
https://nodejs.org/dist/latest-v20.x/docs/api/events.html#eventsgetmaxlistenersemitterortarget
> Added in: v19.9.0

Reply:
https://nodejs.org/docs/latest-v18.x/api/events.html#eventsgetmaxlistenersemitterortarget
> Added in: v18.17.0

v18.17.0 here because v19.9.0 is a higher version than v18.x

Reply:
@peterblazejewicz ok here?